tests/cluster_tests/test_mpiqsub_nosplit.xml

```diff
@@ -16,6 +16,7 @@
     <batchSize>3</batchSize>
     <clusterParameters>-W block=true</clusterParameters>
     <NumThreads>1</NumThreads>
+    <threadCommand>--n-threads=</threadCommand>
```

Comment:
change "--n-threads=" to ""--n-threads"?

Reply:
see below.